packages/core/src/Actor.ts

```diff
@@ -45,8 +45,8 @@ export function createNullActor(id: string): Actor {
  * @param invokeDefinition The meta information needed to invoke the actor.
  */
 export function createInvocableActor<TC, TE extends EventObject>(
-  invokeDefinition: InvokeDefinition<TC, TE>,
-  machine: StateMachine<TC, any, TE>,
+  invokeDefinition: InvokeDefinition<TC, TE, any>,
+  machine: StateMachine<TC, any, TE, any, any>,
```

Comment:
Same here - could `StateMachine` default `any` params to `any` so they don't have to be specified?

Reply:
It already had defaults I've removed those explicit anys from here